src/compojure/api/routes.clj

```diff
@@ -137,12 +139,14 @@
    (reduce
      (fn [acc [path method info]]
        (if-not (true? (:no-doc info))
```

Comment:
Should this be just `(if-not (:no-doc info) ...)`? Or maybe even `(if (:no-doc info) ...)` to avoid double negatives?